application/common/models/User.php

```diff
@@ -411,7 +413,14 @@ public function fields(): array
                 return Utils::getIso8601($model->last_login_utc);
             },
             'manager_email',
-            'personal_email',
+            'personal_email' => function (self $model) {
+                $maskParam = Yii::$app->request->queryParams['mask'] ?? 'yes';
```

Comment:
Is there a use case where `Yii::$app->request` is not present? like in a cron job context?

Reply:
Good question. Yes, it is not present in console context, but the null coalesce operator handles that. I wasn't sure whether it just protected against a missing array key, or if it also protects against a missing object property, so I tested it and it works fine in the console.

Reply:
Nice!